index.html

```diff
@@ -1502,21 +1500,21 @@ <h2>Capabilities</h2>
  </thead>
 
  <tr>
-  <td><dfn>Browser name</dfn>
+  <td>Browser name
```

Comment:
Below we use `<code>` for the individual entries. Is there a reason why that one and others are not wrapped?

```suggestion
  <td><code>Browser name</code>
```

Reply:
I don't agree with this one; they're not code strings, they're just names. The proxy table below is different in that the first column is supposed to represent actual strings. We could make the tables more similar but I think that should be out of scope for this PR.

Reply:
Ah, good point. You are right here.